### DISCUSSION TITLE
13.4.15 listening host issue in docker

### DISCUSSION THREAD

**Discussion by masterkain** [vercel/next.js#54025] [Help] [answered]
I hold back from upgrading from 13.4.12 due to some ENV bugs, tested 13.4.15 but my health checks are failing.

so in standalone mode it seems next server is binding to an ipv6 interface.

---

when the server starts up we can see it's now binding to ipv6 interface.
in the Dockerfile I put `ENV HOSTNAME localhost` to no avail.

13.4.12: `Listening on port 3000 url: http://localhost:3000`
13.4.15: `- ready started server on [::1]:3000, url: http://[::1]:3000`

someone got it working?

**Comment by rafde** [comment]
It seems setting ENV from outside .env file or next.config.js isn't working like it did in 13.4.12. I can't get any of my NEXTAUTH_* ENV values to get read from NextJS either. They print out as `undefined`. The values are coming from AWS Secrets and passed to the docker container. I have to stick to 13.4.12 in the meantime.

**Reply by rafde** [reply]
see https://github.com/vercel/next.js/issues/53367 for why ENV not working

**Comment by masterkain** [comment, marked as answer]
fixed in 13.4.19
